example_project/example_project_combined.json

```diff
@@ -877,6 +881,55 @@
           ]
         ]
       }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "id": "19",
```

Comment:
Adding an ID=19 so we can actually test a SFA (the other SFA is `hpxml_directory`).